README.md

```diff
@@ -22,7 +22,8 @@ MCP is an open-source protocol designed to enable AI models to securely interact
 
 #### Hashicorp
 
-- [Terraform MCP Server](https://github.com/hashicorp/terraform-mcp-server) - Provides seamless integration with Terraform ecosystem, enabling advanced automation and interaction capabilities for Infrastructure as Code (IaC) development. 
+- [Terraform MCP Server](https://github.com/hashicorp/terraform-mcp-server) - Provides seamless integration with Terraform ecosystem, enabling advanced automation and interaction capabilities for Infrastructure as Code (IaC) development.
+- [Vault Radar MCP Server](https://developer.hashicorp.com/hcp/docs/vault-radar/mcp-server/overview) - Provides access to HCP Vault Radar data sources, secret risks, and events, enabling LLMs to query and analyze security information using natural language.
```

Comment:
@gautambaghel can we expect to get a github repo for this?

Reply:
@royderks The server is available for anyone publicly

- DockerHub: https://hub.docker.com/r/hashicorp/vault-radar-mcp-server
- Documentation: https://developer.hashicorp.com/hcp/docs/vault-radar/mcp-server/overview

The team is considering _not_ to make the GitHub repo public because they might add tools in the future which use internal APIs not meant to be exposed publicly.